standalone.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const SerializerSelector = require('./index')
+
+function StandaloneSerializer (options = { readMode: true }) {
+  if (options.readMode === true && !options.restoreFunction) {
```

Comment:
maybe check that restoreFunction is typeof function?

```suggestion
  if (options.readMode === true && typeof options.restoreFunction !== 'function') {
```

Reply:
Did you decided against my remark? Just need feedback that you did not accidently resolved it ;)

Reply:
I think I overwritten the `commit suggestion` during the rebase

Clicking `commit suggestion` mark it as resolved automatically